packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskInputField/components/TaskInputLabel/TaskInputLabel.js

```diff
@@ -7,6 +7,8 @@ import zooTheme from '@zooniverse/grommet-theme'
 import { Markdownz, pxToRem } from '@zooniverse/react-components'
 
 export const StyledTaskInputLabelWrapper = styled(Box)`
+  margin-right: 10px;
```

Comment:
I think this can be set directly on the Box prop using `margin`. https://v2.grommet.io/box#margin